src/servers/mockSocket.js

```diff
@@ -14,7 +14,8 @@ class MockSocketServer extends SocketServer {
 
   bridgeCall(callId, address) {
     const socket = this._getSocket(callId);
-    return Promise.resolve(socket.write(`Bridging ${callId} to ${address}\n`, "utf8"));
+    return Promise.resolve(socket.write(`Bridging ${callId} to ${address}\n`, "utf8"))
+      .then(socket.emit('call.bridged'));
```

Comment:
This works, but looks weird. We don't need to do _then_ here, I think, 'cause we are not dependent on the result of the previous operation.

Actually, we could probably get rid of the whole Promise entirely. Try:
```
bridgeCall(callId, address) {
    const socket = this._getSocket(callId);
    socket.write(`Bridging ${callId} to ${address}\n`, "utf8");
    socket.emit('call.bridged');
}
```
If it works, get rid of the Promises in other methods as well :)